en/releases/README.md

```diff
@@ -1,8 +1,115 @@
+
+For the initial beta release of a new major/minor change, a "branch out deadline" gets set during the Maintainers Call.
+
+The procedure is:
+
+1. Agreement on branch out deadline: By consensus during Maintainers call
+2. Publication in social media & mailing list about the branch out deadline: By [Ramon](https://github.com/mrpollo)
+3. Project board with the name for the new release gets created: By [Daniel](https://github.com/dagar)
+   1. The board must include a `patch version` field (selectable)
+4. Until the branch out date
+   1. All the PR/Issues that a contributor wants to finish before the deadline gets added to the Project Board: By Maintainers & during Maintainers call
+   2. All the release blocking Issues / PRs gets added to the Project Board by each Maintainer
+
+On the branch out date, the `release/Major.minor` branch gets branched out and a `Major.minor.0-beta1` gets released for testing.
+
+#### Beta testing
+
+Before the new major/minor release, the beta version gets released for a testing phase. During the beta testing phase following changes are **allowed**:
+
+- Critical bug fixes
+- New hardware supports
+
+And following changes are **not allowed**:
+
+- Big new features
+- Refactors (with no functional changes)
+
+During the beta testing, the procedure is:
+
+1. During the Maintainers call
+   1. Remaining issues for the next beta release gets discussed, and maintains make an effort to resolve all issues before the next beta release
+   2. Discussion on certain PRs that may be worth to be included in the next beta gets discussed, and gets added to Project Board appropriately
+2. Test reports from users get collected as a Github Issue / Discord message / Flight Review log / Discuss forum post
+   1. If it is valid issue, the maintainer in charge of the sector creates / adds an Issue to keep track of it in the Project tracker and assign the next beta version to it's `patch version` field.
+
+#### Maintainers Agreement before the release
+
+After numerous beta testings & before the official new major/minor release, PX4 Maintainers need to reach a consensus that the new version is release-able from each of their perspectives. This means (for each maintainer):
+
+- The [core functionalities](core_components.md) they maintain has been tested, and has no major issues are found
+- All the outstanding issues/PRs with `patch version` as the last beta version are resolved
+
+#### Point release execution
+
+After the consensus from the Maintainers, the point release gets made:
+
+1. The `patch version` of the last beta release (e.g. `M.m.0-beta9`) gets renamed to include the release name (e.g. `M.m.0/M.m.0-beta9`)
+2. New tag for the last beta release commit gets tagged (as `M.m.0`) by [Daniel](https://github.com/dagar)
+3. Release notes are prepared in Github side collaboratively by Maintainers
+4. Public new point release announcement gets made via Social media / Mailing list by [Ramon](https://github.com/mrpollo)
```

Comment:
I think that in any way the stakeholder should be mentioned in the doc :thinking: 

What I think we could do instead of mentioning individual directly, is we could assign a role of a 'Release maintainer', as referenced by https://github.com/PX4/PX4-user_guide/pull/2313, to be the stakeholder who is in charge of this process.

Or, do you think we don't need to assign the task to anyone? I fear that the responsibilities would be scattered & it would be unclear who is the one working on each pieces of the release procedure